docs/Developers/Executing_Tests.rst

```diff
@@ -56,8 +56,14 @@ To run our tests, you can also use ``python3 setup.py test``
 
 .. note::
 
-    You will not get a test coverage of 100% - the coverage on the
-    website is merged for several python versions.
+    You may not get a test coverage of 100% locally. The coverage
+    published on `codecov.io <https://codecov.io/gh/coala/>`__ (GitHub
+    Projects) and `codecov.io <https://codecov.io/gl/coala/>`__ (GitLab
+    Projects) are actually merged results for several python versions.
+    Also, the results are merged from different OS. Appveyor results
+    provide coverage of Windows specific lines, and Travis/Circle
+    provide coverage of Unix specific lines. Thus, your test coverage
+    can 'pass' without reaching 100% .
```

Comment:
should help if we briefly mention the fact that the person solving the issue should concentrate on getting 100% coverage on the changes that he/she is making rather than worrying about the coverage report of the whole project.